consul/map.jinja

```diff
@@ -10,8 +10,7 @@
         'service': 'consul',
         'conf_file': '/etc/consul/consul.conf',
         'config_dir': '/etc/consul.d',
-        'data_dir': '/var/lib/consul',
-        'dns_port': 8600
```

Comment:
Might want to keep this and default it to port 53

Reply:
It's not being used anywhere as the consul config file is generated from pillar data

Reply:
Fair enough. I was just thinking we should maybe update the formula to provide that as a default.